### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@vikivivi please review at your convenience, TIA!

**Comment by luzpaz** [comment]
bump

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -17944,6 +17950,7 @@ implemnt->implement
 implemntation->implementation
 implemntations->implementations
 implemtation->implementation
+implemtations->implementations
 implemtentation->implementation
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:17954]:
```suggestion
implemt->implement
implemtation->implementation
implemtations->implementations
implemted->implemented
implemtentation->implementation
implemtentations->implementations
implemting->implementing
implemts->implements
```
* https://github.com/search?q=implemt&type=code  39000
* https://github.com/search?q=implemtation&type=code  43468
* https://github.com/search?q=implemtations&type=code  4862
* https://github.com/search?q=implemted&type=code  29060
* https://github.com/search?q=implemtentation&type=code  596
* https://github.com/search?q=implemtentations&type=code  51
* https://github.com/search?q=implemting&type=code  2155
* https://github.com/search?q=implemts&type=code  693

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21125,6 +21140,9 @@ meaninless->meaningless
 meaninng->meaning
 mear->wear, mere, mare,
 mearly->merely, nearly,
+meassure->measure
+meassured->measured
+meassures->measures
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:21145]:
```.text
meassurable->measurable
meassurably->measurably
meassurement->measurement
meassurements->measurements
meassuring->measuring
```
* https://github.com/search?q=meassurable&type=code  252
* https://github.com/search?q=meassurably&type=code  3
* https://github.com/search?q=meassurement&type=code  5138
* https://github.com/search?q=meassurements&type=code  2786
* https://github.com/search?q=meassuring&type=code  42076

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21822,6 +21840,7 @@ Mocrosoft->Microsoft
 mocule->module
 mocules->modules
 moddel->model
+moddeled->modeled
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:21843]: Shall we also include `en-GB` variant?
```text
moddelled->modelled
```
* https://github.com/search?q=moddeled&type=code  231
* https://github.com/search?q=moddelled&type=code  47

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -35140,6 +35164,7 @@ unesacpe->unescape
 unesacped->unescaped
 unessecarry->unnecessary
 unessecary->unnecessary
+unevaluted->unevaluated
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:35167]:
```text
evalutions->evaluations
evalutive->evaluative
evalutor->evaluator
evalutors->evaluators
```
* https://github.com/search?q=unevaluted&type=code  2952
* https://github.com/search?q=evalutions&type=code  5838
* https://github.com/search?q=evalutive&type=code  20
* https://github.com/search?q=evalutor&type=code  23311
* https://github.com/search?q=evalutors&type=code  1160

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -16774,6 +16779,7 @@ greif->grief
 grephic->graphic
 grestest->greatest
 greysacles->greyscales
+gride->grid
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: removed. thanks!

**Comment by vikivivi** [comment]
LGTM. (Approving will be done by reviewers with write access. It wasn't me)

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**